[2025-02-11T08:59:40.427+0600] {processor.py:186} INFO - Started process (PID=5640) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T08:59:40.428+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T08:59:40.431+0600] {logging_mixin.py:190} INFO - [2025-02-11T08:59:40.431+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T08:59:40.435+0600] {logging_mixin.py:190} INFO - [2025-02-11T08:59:40.434+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T08:59:40.436+0600] {logging_mixin.py:190} INFO - [2025-02-11T08:59:40.435+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T08:59:40.436+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T08:59:40.451+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2025-02-11T09:01:03.554+0600] {processor.py:186} INFO - Started process (PID=5925) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:01:03.560+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T09:01:03.563+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:01:03.563+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:01:03.568+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:01:03.567+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T09:01:03.568+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:01:03.568+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T09:01:03.568+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:01:03.581+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2025-02-11T09:02:28.530+0600] {processor.py:186} INFO - Started process (PID=6332) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:02:28.538+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T09:02:28.544+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:02:28.543+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:02:28.555+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:02:28.553+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T09:02:28.556+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:02:28.556+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T09:02:28.557+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:02:28.575+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2025-02-11T09:03:54.405+0600] {processor.py:186} INFO - Started process (PID=6811) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:03:54.412+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T09:03:54.417+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:03:54.416+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:03:54.426+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:03:54.425+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T09:03:54.427+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:03:54.426+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T09:03:54.427+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:03:54.453+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2025-02-11T09:05:10.388+0600] {processor.py:186} INFO - Started process (PID=7113) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:05:10.394+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T09:05:10.396+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:05:10.395+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:05:10.401+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:05:10.400+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T09:05:10.401+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:05:10.401+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T09:05:10.401+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:05:10.415+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2025-02-11T09:06:30.375+0600] {processor.py:186} INFO - Started process (PID=7471) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:06:30.393+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T09:06:30.396+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:06:30.395+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:06:30.402+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:06:30.401+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T09:06:30.403+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:06:30.403+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T09:06:30.403+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:06:30.423+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
[2025-02-11T09:07:53.386+0600] {processor.py:186} INFO - Started process (PID=7985) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:07:53.392+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T09:07:53.394+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:07:53.393+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:07:53.400+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:07:53.399+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T09:07:53.400+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:07:53.400+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T09:07:53.400+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:07:53.416+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2025-02-11T09:09:20.402+0600] {processor.py:186} INFO - Started process (PID=8510) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:09:20.404+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T09:09:20.406+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:09:20.405+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:09:20.413+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:09:20.412+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T09:09:20.414+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:09:20.414+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T09:09:20.415+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:09:20.430+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2025-02-11T09:10:41.176+0600] {processor.py:186} INFO - Started process (PID=8988) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:10:41.182+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T09:10:41.184+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:10:41.184+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:10:41.190+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:10:41.189+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T09:10:41.191+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:10:41.191+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T09:10:41.191+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:10:41.206+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2025-02-11T09:12:04.199+0600] {processor.py:186} INFO - Started process (PID=9505) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:12:04.206+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T09:12:04.210+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:12:04.209+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:12:04.221+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:12:04.219+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T09:12:04.222+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:12:04.222+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T09:12:04.223+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:12:04.255+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2025-02-11T09:13:25.530+0600] {processor.py:186} INFO - Started process (PID=10038) to work on /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:13:25.538+0600] {processor.py:914} INFO - Processing file /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T09:13:25.542+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:13:25.540+0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:13:25.553+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:13:25.551+0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T09:13:25.554+0600] {logging_mixin.py:190} INFO - [2025-02-11T09:13:25.554+0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T09:13:25.555+0600] {processor.py:927} WARNING - No viable dags retrieved from /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T09:13:25.584+0600] {processor.py:208} INFO - Processing /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
